content/docs/rendering-elements.md

```diff
@@ -1,75 +1,75 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Renderowanie elementów
 permalink: docs/rendering-elements.html
 redirect_from:
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Elementy to najmniejsze bloki budujące reactowe aplikacje.
 
-An element describes what you want to see on the screen:
+Element opisuje co chcesz zobaczyć na ekranie.
 
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+W przeciwieństwie do elementów drzewa DOM w przeglądarce, reactowe elementy są zwykłymi obiektami i mają niski koszt stworzenia.
 
->**Note:**
+>**Wskazówka:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>Łatwo pomylić elementy z szerzej znaną koncepcją "komponentów". Komponenty przedstawimy w [kolejnej sekcji](/docs/components-and-props.html). Elementy są tym, z czego komponenty "są zbudowane". Zachęcamy do przeczytania tej sekcji przed przejściem dalej.
 
-## Rendering an Element into the DOM {#rendering-an-element-into-the-dom}
+## Renderowanie elementu w drzewie DOM {#rendering-an-element-into-the-dom}
 
-Let's say there is a `<div>` somewhere in your HTML file:
+Powiedzmy, że gdzieś w twoim pliku HTML jest `<div>`:
 
 ```html
 <div id="root"></div>
 ```
 
-We call this a "root" DOM node because everything inside it will be managed by React DOM.
+Nazywamy ten węzeł drzewa DOM "korzeniem", bo wszytko, co się w nim znajduje będzie zarządzane przez React DOM.
```

Comment:
```suggestion
Ten węzeł nazywamy "korzeniem" drzewa DOM, bo wszystko, co się w nim znajduje, będzie zarządzane przez React DOM.
```

Reply:
Jeśli chodzi o pierwsze zdanie, to wydaje mi się że nie chodzi o korzeń drzewa DOM tylko o korzeń Reactowego drzewa, którym jest ten węzeł, więc poprawka zmienia trochę znaczenie tego zdania. Sugerowałbym może coś takiego:
```suggestion
Ten węzeł drzewa DOM nazywamy "korzeniem", bo wszytko, co się w nim znajduje będzie zarządzane przez React DOM.
```

Reply:
Masz rację, źle zrozumiałem to zdanie :)

Reply:
Ok, w takim razie zamykam